src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const items = document.querySelectorAll('.population');
+
+const populations = [...items].map(item => item.innerText.split(',').join(''));
+
+const result = populations.reduce((sum, item) => sum + Number(item), 0);
```

Comment:
```suggestion
const result = populations.reduce((sum, item) => sum + +item, 0);
```

Reply:
I've tried deploying the demo links but it always throws an error. Tell me what to do?